pkg/promlib/admission_handler.go

```diff
@@ -48,8 +48,8 @@ func (s *Service) MutateAdmission(ctx context.Context, req *backend.AdmissionReq
 	default:
 		return getBadRequest(fmt.Sprintf("expected apiVersion: v0alpha1, found: %s", settings.APIVersion)), nil
 	}
-	if settings.URL != "" {
-		return getBadRequest("unsupported URL value"), nil
+	if settings.URL == "" {
```

Comment:
Nit. Maybe should default to http://localhost:9090 🤔

Reply:
FWIW, this error will never be returned IRL because there is a previous check for core datasources that check if the URL is empty ([here](https://github.com/grafana/grafana/blob/admissionHandlerFix/pkg/api/datasources.go#L568))